src/main/resources/structure/kettle.properties

```diff
@@ -0,0 +1,5 @@
+ETL_SOURCE_DATABASE=fgh
```

Comment:
lets rename the `structure` folder to be a little more descriptive. Maybe something like `pdiresources` and then within that folder I'd like to see two sub-folders: `transformations` and `jobs`

Reply:
done